x-pack/legacy/plugins/ml/server/models/data_recognizer/modules/apm_jsbase/ml/datafeed_abnormal_span_durations_jsbase.json

```diff
@@ -0,0 +1,31 @@
+{
+  "job_id": "JOB_ID",
+  "indices": [
+    "INDEX_PATTERN_NAME"
+  ],
+  "max_empty_searches": 10,
+  "query": {
+    "bool": {
+      "must": [
+        {
```

Comment:
We try and use single line JSON where possible for each filter term to improve readability. so that this could be something like:

```
"must": [
  { "bool": { "filter": { "term": { "agent.name": "js-base" } } } },
  { "bool": { "filter": { "term": { "processor.event": "span" } } } }
]
```

Reply:
https://github.com/elastic/kibana/pull/53792/commits/f40d0a205dc8e20394e6900b6f98946d857a5b69